Makefile

```diff
@@ -189,8 +189,8 @@ install-crds: manifests
 	kubectl apply -f deploy/crds/policy.open-cluster-management.io_policysets.yaml
 	kubectl apply -f https://raw.githubusercontent.com/open-cluster-management/multicloud-operators-placementrule/main/deploy/crds/apps.open-cluster-management.io_placementrules_crd.yaml
 	kubectl apply -f https://raw.githubusercontent.com/open-cluster-management/api/main/cluster/v1/0000_00_clusters.open-cluster-management.io_managedclusters.crd.yaml
-	kubectl apply -f https://raw.githubusercontent.com/open-cluster-management/api/main/cluster/v1alpha1/0000_03_clusters.open-cluster-management.io_placements.crd.yaml
```

Comment:
Looks like there might be some other URLs pointing to `open-cluster-management` here also. Would you please update those also?